spec.bs

```diff
@@ -7657,6 +7672,9 @@ An <dfn>auction data config</dfn> is a [=struct=] with the following [=struct/it
   :  <dfn>coordinator</dfn>
   :: The origin of the coordinator hosting public encryption keys for the server
     running the ad auction. The [=origin/scheme=] must be "`https`".
+  :  <dfn>encryption key</dfn>

```

Comment:
https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#section-2.2.4 takes "the HPKE public key and its associated key ID" as an input parameter. Does this contain the key ID?